lib/server.js

```diff
@@ -123,6 +142,16 @@ function create(log, error, config, routes, db) {
       }
     )
     server.auth.strategy(
+      // This strategy fetches the sessionToken with its
+      // verification state. It doesn't check that state.
+      'sessionTokenVerified',
```

Comment:
> My understanding is that adding new auth strategies will require updates to clients to create Hawk credentials using `sessionTokenVerified` instead of `sessionToken`.

Oh really? I had no idea that the name of the strategy was significant in that way, but it totally makes sense now you mention it. Is there a way to tell the strategy to use a specific field name? (I'll look into that now, just asking in case you know the answer)

Failing that, the choice would be between forcing all `sessionToken` requests to join with the `unverifiedTokens` table or querying the database twice whenever we need the verification state. Presumably the second of those would be the lesser of two evils but they're both sub-optimal.